### PR TITLE
Implement database to store private chain data

### PR DESCRIPTION
**Pull request by t-lichtenstein**

### CODE REVIEW

File: cassandra/create.cql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+CREATE KEYSPACE ethereum WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};
+CREATE TABLE messages(
+    username text,
+    sent_at timestamp,
+    message text,
+    sender text,
+    PRIMARY KEY(username, sent_at)
+);
```
Comment (simsieg) [inline cassandra/create.cql]: Delete this folder

File: src/mongoConnector.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    .then(() => {
+      log.info('Mongodb has been connected')
+    })
+    .catch(error => {
+      log.info('Error while trying to connect with mongodb')
+      throw error
+    })
+
+  return mongoDB
+}
```
Comment (fyndalf) [inline src/mongoConnector.js]: insert trailing newline

File: components/privateChains/backendListener.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      log.info('Change Buffer to Buffer B')
+    }
+    else {
+      CurrentBuffer = BufferA
+      log.info('Change Buffer to Buffer A')      
+    }
+    isBufferA = !isBufferA
+  }, 15000) 
+
+  // const currentBuffer = BufferA
```
Comment (simsieg) [inline components/privateChains/backendListener.js]: Remove commented code

**Review by fyndalf — CHANGES_REQUESTED** [review]

File: components/privateChains/backendListener.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+          }
+        })
+      } 
+      catch (error) {
+        log.info(error)
+        socket.send(415)
+      } 
+    })
+  })
+}
```
Comment (simsieg) [inline components/privateChains/backendListener.js]: Insert trailing newline

File: src/mongoConnector.js (hunk 2 above)
Comment (simsieg) [inline src/mongoConnector.js]: Trailing newline

**Review by simsieg — CHANGES_REQUESTED** [review]

**Review by fyndalf — APPROVED** [review]

**Review by simsieg — APPROVED** [review]

**Pull request merged**